src/components/WebexAvatar/WebexAvatar.test.js

```diff
@@ -1,19 +1,41 @@
 import React from 'react';
 
+import PeopleJSONAdapter from '../../adapters/PeopleJSONAdapter';
+import {personStatus} from '../../adapters/PeopleAdapter';
+import person from '../../data/person';
+
 import WebexAvatar from './WebexAvatar';
 
 describe('Webex Avatar component:', () => {
-  let wrapper;
+  let personID, peopleJSONAdapter;
 
   beforeEach(() => {
-    wrapper = shallow(<WebexAvatar />);
+    [personID] = Object.keys(person);
+    peopleJSONAdapter = new PeopleJSONAdapter(person);
+    global.console.error = jest.fn();
+  });
+
+  test('matches snapshot with "default" status', () => {
+    expect(shallow(<WebexAvatar personID={personID} adapter={peopleJSONAdapter} />)).toMatchSnapshot();
   });
 
-  test('exists', () => {
-    expect(wrapper).toMatchSnapshot();
+  Object.keys(personStatus).forEach((status) => {
```

Comment:
I'm really not a fan of these test generators in the SDK. I'd prefer there being a test case for each.

Reply:
Agreed, while is easier to code, it's much harder to read later. It is also counter to "tests as documentation" way of thinking.